yogstation/code/modules/mob/mob_movement.dm

```diff
@@ -0,0 +1,50 @@
+#define MAX_SW_LUMS 0.2
+#define ALLOW_PULL_THROUGH_WALLS 0
+#define SW_LIGHT_FACTOR 2.75
+
+/proc/Can_ShadowWalk(var/mob/mob)
```

Comment:
Not gonna review this file because I don't like it, see earlier review comments.